schema.graphql

```diff
@@ -267,6 +267,12 @@ type Account @entity {
   currencyBalances: [CurrencyBalance] @derivedFrom(field: "account")
 }
 
+type EvmAccount @entity {
+  id: ID! #Address
```

Comment:
Which address is this? If just the EVM address, that wouldn't work because it would be the same on different chain IDs.

I think we should probably say `ID = chainId + address`, and add a separate `address: String! @index` param or something like that

Reply:
Will tackle that in the next release!